docs/en/sql-reference/table-functions/executable.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: /en/engines/table-functions/executable
+sidebar_position: 55
+sidebar_label:  executable
+keywords: [udf, user defined function, clickhouse, executable, table, function]
+---
+
+# executable Table Function for UDFs
+
+The `executable` table function creates a table based on the output of a user-defined function (UDF) that you define in a script that outputs rows to **stdout**. The executable script is stored in the `users_scripts` directory and can read data from any source.
```

Comment:
The biggest difference between `UDF functions` and `executable table function`, `Executable`, `ExecutablePool` storages is that ordinary `UDF function` cannot change row count. If it has input of 15 rows it must return 15 rows. But `executable table function`, `Executable`, `ExecutablePool` can make any data transformations for example complex aggregations.